spec.md

```diff
@@ -3574,17 +3598,18 @@ module = "module" , id , ":" , [ info ] , newline , indent ,
            { port , newline } ,
            { statement , newline } ,
          dedent ;
+type_param = int | string | raw_string ;
```

Comment:
i don't get the difference between string and raw_string other than how they are emitted Didn't we also get rid of the RawString version of this from chisel in terms of the ExtModule use of it? Can we just get rid of raw_string? Is the only point of it to let you put `"` inside the string...?

Reply:
Here's a small circuit:

```
circuit Foo:
  extmodule Bar:
    output a: UInt<1>
    parameter foo = 'hello'
    parameter bar = "world"

  module Foo:
    output a: UInt<1>

    inst bar of Bar
    a <= bar.a
```

This compiles to Verilog as:
```
```

Reply:
ok. but again i thought we got rid of the RawStringParam Chisel API so is there still a use for this?